l10n/en/navigation_refresh.ftl

```diff
@@ -22,6 +22,7 @@ navigation-refresh-innovation-projects-v2 = Innovation Projects
 navigation-refresh-blog = Blog
 navigation-refresh-our-mission = Our Mission
 navigation-refresh-our-work = Our Work
+navigation-refresh-mozilla-builders = { -brand-name-mozilla } Builders
```

Comment:
I don't think we want "Builders" to be translated, so it's best to add a Brand string for "Mozilla Builders" under Mozilla projects here: https://github.com/mozilla/bedrock/blob/main/l10n/en/brands.ftl#L109

Then we can use that brand identifier here